env.template

```diff
@@ -60,15 +60,16 @@ MANAGER_IP="$(hostname -I | cut -d' ' -f1)"
 # Set to OAUTH if using OAuth Password Flow Authentication, callback url needs to be api/v2/results
 CALLBACK_AUTHENTICATION=TOKEN
 
-CLIENT_ID=sensor01.sms.internal
+CLIENT_ID=sensor01.sms.internal # must match FQDN
```

Comment:
Can you not just use socket.getfqdn(). Will that not work in Docker?

Reply:
Not able to get the right value in docker